frontend/lib/Views/CompareEndpointsView.dart

```diff
@@ -22,91 +23,93 @@ class _CompareChartsViewState extends State<CompareChartsView> {
+                data: ThemeData.from(
+                    colorScheme: ColorScheme.fromSwatch(
+                        backgroundColor: Colors.white,
+                        cardColor: Colors.pink,
+                        primarySwatch: Colors.pink)),
+                child: DropDownMultiSelect(
+                  isDense: true,
+                  options: endpointModel.endpointSummaryMap.keys.toList(),
+                  selectedValues: endpointModel.selectedEndpoints,
+                  onChanged: (List<String> selected) {
+                    setState(
+                      () {
+                        endpointModel.updateEndpointSelectedList(selected);
                       },
-                    ),
-                  ],
+                    );
+                  },
+                  whenEmpty: emptyField,
                 ),
               );
-            }
-          },
-        ),
-      ),
-    );
-
-  Row buildDropDownSelection(
-      BuildContext context, AsyncSnapshot<List<EndpointSummary>> snapshot) => Row(
-      mainAxisAlignment: MainAxisAlignment.spaceAround,
-      children: <Widget>[
-        SizedBox(
-          width: MediaQuery.of(context).size.width * 0.8,
-          child: Consumer<CompareEndpointsModel>(
-              builder: (context, endpointModel, _) {
-            endpointModel.makeEndpointSummaryMap(snapshot.data!);
-            return Theme(  // 200IQ move
-              data: ThemeData.from(colorScheme: ColorScheme.fromSwatch(
-                backgroundColor: Colors.white,
-                cardColor: Colors.pink,
-                primarySwatch: Colors.pink
-              )),
-              child: DropDownMultiSelect(
-                isDense: true,
-                options: endpointModel.endpointSummaryMap.keys
-                    .toList(),
-                selectedValues: endpointModel.selectedEndpoints,
-                onChanged: (List<String> selected) {
-                  setState(
-                    () {
-                      endpointModel.updateEndpointSelectedList(selected);
-                    },
-                  );
-                },
-                whenEmpty: emptyField,
-              ),
-            );
-          }),
-        ),
-      ],
-    );
-
-  List<Widget> _createChips(CompareEndpointsModel endpointModel) => endpointModel.commonFields.map((e) {
-      endpointModel.selectedChips.putIfAbsent(e, () => false);
-      return Container(
-        margin: const EdgeInsets.symmetric(horizontal: 8, vertical: 0),
-        child: InputChip(
-          label: Text(e),
-          selected: endpointModel.selectedChips[e]!,
-          onSelected: (bool value) {
-            setState(() {
-              endpointModel.selectChips(e, value);
-            });
-          },
-        ),
+            }),
+          ),
+        ],
       );
-    }).toList();
+
+  List<Widget> _createChips(CompareEndpointsModel endpointModel) =>
+      endpointModel.commonFields.map((e) {
```

Comment:
To `e` chyba nie za bardzo mówi czym jest. Może zmienilibyśmy nazwę na `commonField`, albo `field`?